pkg/bundle/bundle.go

```diff
@@ -215,13 +220,30 @@ func (b *bundle) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result,
 		Message: message,
 	}
 
+	// Requeue after the minimum poll interval time of all HTTPS bundles
+	requeueAfter := time.Duration(0)
```

Comment:
This looks like to me that requeueAfter will be 0 if there are no HTTPS sources. I _think_ that is fine because a requeueAfter value of 0 is ignored in controller-runtime (?). Can you add a comment to that effect?